test/get_test.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2012 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Unit tests for get.
+ * @author bnkuhn@gmail.com (Brian Kuhn)
```

Comment:
Maybe add yourself here? I've just been removing this comment. If you want you can just leave it here, I'm not sure how authorship works when you translate something.